scripts/approvalYears.py

```diff
@@ -222,15 +225,15 @@ def apprDateRegression(prods):
                 datays[kind2].append(ts)
 
     models = dict()
-    for kind in dataxs.keys():
+    for kind in list(dataxs.keys()):
```

Comment:
Same as above: "for kind in dataxs:"

Reply:
Python3 versus python2 issue. 2to3 automagically has made this change as .keys() no longer returns a list in python3